examples/go-code-tutorial/cmd/roll.go

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Please set a consistent license header everywhere (see https://github.com/bazelbuild/rules_go/blob/d517cd9bec43f0541b421315fe633cdd0f204270/go/tools/bazel/runfiles.go#L1). https://github.com/google/addlicense makes this easy.